### ISSUE TITLE
CBA Settings dependend postInits should run via `CBA_settingsInitialized` event

### ISSUE THREAD

**Issue by veteran29** [ArmaForces/Mods#99]
- title

CBA Settings values are not guaranteed to be already sent from server to client in client postInit. Code that is depended on them and initializes only once should wait for `CBA_settingsInitialized `.

Ref: https://github.com/CBATeam/CBA_A3/blob/master/addons/settings/XEH_postInit.sqf#L3
Might be useful: https://github.com/zen-mod/ZEN/blob/86fc60e3a8aaa2eff6e4243e2f1e9d5121712719/addons/common/XEH_preInit.sqf#L18

**Comment by 3Mydlo3** [comment]
Until now we didn't encounter any issues related to this so I think it's not necessary to "fix" it right away. Still it's nice to have it here and in case of some issues, we might find this as worth fixing.

**Comment by veteran29** [comment]
We didn't had any issues or we did not notice, it can be quire rare, can happen only for some clients and is an issue only for things that initialize once, and we do not have a lot of these on clientside. Most of the things initializing once are serverside I belive and issue is not present there.
